src/config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "numHighlightedBuckets": 10,
```

Comment:
Note to self: Use the config/ directory instead, maybe config/firefox-desktop.js

Reply:
NTS: Actually, create a file config/shared.js